# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mozplugger/Makefile.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2005-08-29 21:09:55.000000000 +0000
+++ ./Makefile	2006-02-20 15:56:43.000000000 +0000
@@ -99,7 +99,7 @@
 NORM_LDFLAGS=
 XCFLAGS=-shared
 XLDFLAGS=-shared
-XLIBDIR = $(X11)/lib
+XLIBDIR = $(X11)/lib${libsuff}
 XLIBS=-lX11 -L$(XLIBDIR)
 
 LDSHARED=$(LD) $(XLDFLAGS)

